### ISSUE TITLE
Steps Limit In Games

### ISSUE THREAD

**Issue by KarlXing** [openai/procgen#34]
Hi, I have a question about the steps limit in games. For example, I created a venv of 'coinrun' from ProcgenEnv and the episode will be finished if the agent didn't get the reward within 1000 steps. I cannot tell the episode is finished because the agent was killed by an enemy or goes beyond the steps limit from the info returned after taking one step. 

Another thing is I'm trying to use a fixed sequence of levels for one game like
`ProcgenEnv(num_envs=6, env_name="coinrun", start_level=0, num_levels=1, use_sequential_levels=True)`. It should start from level 0 and goes to next level once a level is passed. In this way, reaching the end of a level does not end the episode. May I know if the steps limit of 1000 will apply to a single level here or for the whole episode?

I'm also curious about why we need to set `num_levels=1` if we want to have a single linear series of levels similar to a gym-retro or ALE game according to the ['Environment Options' section in readme](https://github.com/openai/procgen#environment-options). What will happen if we set `num_levels=0`? Will the next level be generated randomly? 

In summary, I have three questions.  
1. How to tell if the episode is done because of time limit or not? 
2. Will the time limit affect a single level or the whole episode when I use a fixed sequence of levels for a ALE-like game.
3. Why we need to set `num_levels=1` if we want to have a single linear series of levels similar to a gym-retro or ALE game.

Thank you in advance for creating this benchmark and your help!

**Comment by KarlXing** [comment]
Okay, I found the steps limit is defined as timeout in [game.cpp](https://github.com/openai/procgen/blob/master/procgen/src/game.cpp). My left question is when I created a fixed sequence of levels. Will the timeout parameter affect a single level or the whole episode? Thank you.

**Comment by christopherhesse** [comment]
1) I believe you could add a new info field if you wanted to know if you died on the last timestep of a level, see my answer to this other issue for hints on how to do that: https://github.com/openai/procgen/issues/32#issuecomment-589868469

2) It looks like the steplimit is per-level

3) Let's say you set `start_level=0`, `num_levels=2`, that means that each time the agent dies it gets a level with seed `0` or `1`.  `use_sequential_levels` adds a fixed number to that seed to get each subsequent level.  In ALE style games there is only a single chain of levels, so to mirror that you should have `num_levels=1`.  `num_levels=2` would produce 2 chains of levels, and `num_levels=0` would produce a very large number of chains.

**Comment by infinity2357** [comment]
Is there a way to increase the steps limit of 1000 for episodes before termination?
